robot_smach_states/CMakeLists.txt

```diff
@@ -14,5 +14,7 @@ catkin_python_setup()
 catkin_package()
 
 if (CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
   catkin_add_nosetests(test)
+  add_rostest(test/rostest/test_grab.test)
```

Comment:
For some reason I did not need to separate rostests from nosetests like in robot_skills. I dont know why though

Reply:
Probably because there is no`__init__.py` in the `rostest` folder